src/rules/sort-imports.md

```diff
@@ -23,36 +21,36 @@ import {foo, bar} from "my-module.js";
 import * as myModule from "my-module.js";
 ```
 
-The import statement can also import a module without exported bindings. Used when the module does not export anything, but runs it own code or changes the global context object.
+import 语句也可以导入一个没有导出绑定的模块。当模块不输出任何东西，但运行它自己的代码或改变全局上下文对象时使用。
 
 ```js
 // none - Import module without exported bindings.
 import "my-module.js"
 ```
 
-When declaring multiple imports, a sorted list of import declarations make it easier for developers to read the code and find necessary imports later. This rule is purely a matter of style.
+当声明多个导入时，导入声明的分类列表使开发者更容易阅读代码，并在以后找到必要的导入。这条规则纯粹是一个风格问题。
 
-## Rule Details
+## 规则细节
 
-This rule checks all import declarations and verifies that all imports are first sorted by the used member syntax and then alphabetically by the first member or alias name.
+这个规则检查所有的导入声明，并验证所有的导入首先按照使用的成员语法排序，然后按照第一个成员或别名的字母顺序排序。
 
-The `--fix` option on the command line automatically fixes some problems reported by this rule: multiple members on a single line are automatically sorted (e.g. `import { b, a } from 'foo.js'` is corrected to `import { a, b } from 'foo.js'`), but multiple lines are not reordered.
+命令行中的 `--fix` 选项会自动修复本规则报告的一些问题：单行的多个成员会被自动排序（例如，`import { b, a } from 'foo.js'` 被修正为 `import { a, b } from 'foo.js'`），但多行不会被重新排序。
 
-## Options
+## 选项
 
-This rule accepts an object with its properties as
+该规则接受一个对象，其属性为
 
-* `ignoreCase` (default: `false`)
-* `ignoreDeclarationSort` (default: `false`)
-* `ignoreMemberSort` (default: `false`)
-* `memberSyntaxSortOrder` (default: `["none", "all", "multiple", "single"]`); all 4 items must be present in the array, but you can change the order:
-    * `none` = import module without exported bindings.
-    * `all` = import all members provided by exported bindings.
-    * `multiple` = import multiple members.
-    * `single` = import single member.
-* `allowSeparatedGroups` (default: `false`)
+* 忽略大小写（默认：`false`)
+* 忽略声明排序 (ignoreDeclarationSort)（默认：`false`)
+* 忽略成员排序 (ignoreMemberSort)（默认为 `false`)
```

Comment:
I'm guessing this was accidentally translated,  since these are property names in the configuration object so they should probably stay as-is.